src/Processors/Merges/Algorithms/AggregatingSortedAlgorithm.h

```diff
@@ -23,8 +25,69 @@ class AggregatingSortedAlgorithm final : public IMergingAlgorithmWithDelayedChun
     void consume(Input & input, size_t source_num) override;
     Status merge() override;
 
-    struct SimpleAggregateDescription;
-    struct AggregateDescription;
```

Comment:
How did it work before 😮?

Reply:
I don't fully understand it by myself but I think it has to do with slightly different inlining with libcxx 15.

The problem is even worse with clickhouse-copier ... it fell into thousands of pieces and I try to fix it.

Reply:
I think it was possible to declare `std::vector<AggregateDescription>` cause you did not need to know `sizeof(AggregateDescription)` in order to calculate `sizeof(std::vector<AggregateDescription>)`

Reply:
that's the interesting part for me, why do they need size now 🤔

Reply:
Looks like it had been introduced here - https://github.com/llvm/llvm-project/commit/f4fb72e6d4cee1097e6606e66232fe55e793cd86